scripts/run-pep8.sh

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env bash
 
-MAX_PEP8_VIOLATIONS=110
+MAX_PEP8_VIOLATIONS=111
 
 mkdir -p test/logs
 PEP8_VIOLATIONS=test/logs/pep8.txt
 touch $PEP8_VIOLATIONS
 
-pep8 --config=.pep8 openassessment > $PEP8_VIOLATIONS
+pep8 --config=.pep8 openassessment test > $PEP8_VIOLATIONS
```

Comment:
Fun fact: everything in the `test` directory was not being included in quality tests. I've added it here and updated the violations numbers to match.